vipermonkey/core/__init__.py

```diff
@@ -242,19 +242,23 @@ def add_compiled_module(self, m):
             return
         self.modules.append(m)
         for name, _sub in m.subs.items():
-            log.debug('(1) storing sub "%s" in globals' % name)
+            if (log.getEffectiveLevel() == logging.DEBUG):
```

Comment:
@kirk-sayre-work, just curious: why do you need to check log.getEffectiveLevel? Normally this is exactly what log.debug does, it only logs the message if the logging level matches DEBUG.